templates/gitlab/mender-artifact-build.gitlab-ci.yml

```
@@ -0,0 +1,24 @@
+# GitLab CI template to build Mender artifacts.
+#
+# The template is designed to be used with the includes keyword or copied into an existing
+# configuration. It can be used to build a Mender artifact in the `script` section. 
+# `before_script` is used for creating the `MENDER_ARTIFACTS_FOLDER` folder. If it's overwritten,
+# then make sure your script contains this step.
+#
+# Mender artifacts in the `MENDER_ARTIFACTS_FOLDER` folder will be uploaded as GitLab CI artifacts
+# and shared across jobs in a pipeline.
+
+include:
+  - remote: "https://raw.githubusercontent.com/mendersoftware/mender-ci-workflows/master/templates/gitlab/common.gitlab-ci.yml"
+
+.build:mender:artifact:
```

For naming (applies to all jobs defined in the GitLab templates), I would use the convention `verb:noun` like:
```suggestion
.build:mender-artifact:
```

Otherwise it gets confusing as "mender" or "artifact" are not really semantically in the same group as "build"

---

Another option, maybe better, is to have all the jobs with `.mender:` prefix. So that you everything you include from a template is in the same "namespace". In this case this job could be:
```suggestion
.mender:build:artifact:
```
or
```suggestion
.mender:build:mender-artifact:
```